modules/space/rendering/renderablefluxnodes.cpp

```diff
@@ -0,0 +1,1106 @@
+
+        //not in use atm.
+        _shaderProgram->setUniform("modelViewProjection",
+            data.camera.sgctInternal.projectionMatrix() * glm::mat4(modelViewMat)
+        );
+
+        SceneGraphNode* earthNode = sceneGraphNode("Earth");
+        if (!earthNode) {
+            LWARNING("Could not find scene graph node 'Earth'.");
+        }
+        glm::vec3 earthPos = earthNode->worldPosition() * data.modelTransform.rotation;
+    
+        _shaderProgram->setUniform(_uniformCache.streamColor, _streamColor);
+        _shaderProgram->setUniform(_uniformCache.nodeSize, _nodeSize);
+        _shaderProgram->setUniform(
+            _uniformCache.nodeSizeLargerFlux, 
+            _nodeSizeLargerFlux
+        );
+        _shaderProgram->setUniform(_uniformCache.thresholdFlux, _thresholdFlux);
+        _shaderProgram->setUniform(_uniformCache.colorMode, _colorMode);
+        _shaderProgram->setUniform(_uniformCache.filterLower, _filteringLower);
+        _shaderProgram->setUniform(_uniformCache.filterUpper, _filteringUpper);
+        _shaderProgram->setUniform(_uniformCache.scalingMode, _scalingmethod);
+        _shaderProgram->setUniform(
+            _uniformCache.colorTableRange, 
+            _colorTableRange.value()
+        );
+        _shaderProgram->setUniform(_uniformCache.domainLimZ, _domainZ.value());
+        _shaderProgram->setUniform(_uniformCache.nodeSkip, _amountofNodes);
+        _shaderProgram->setUniform(_uniformCache.nodeSkipDefault, _defaultNodeSkip);
+        _shaderProgram->setUniform(_uniformCache.nodeSkipEarth, _earthNodeSkip);
+        _shaderProgram->setUniform(_uniformCache.nodeSkipMethod, _nodeskipMethod);
+        _shaderProgram->setUniform(
+            _uniformCache.nodeSkipFluxThreshold, 
+            _fluxNodeskipThreshold
+        );
+        _shaderProgram->setUniform(
+            _uniformCache.nodeSkipRadiusThreshold, 
+            _radiusNodeSkipThreshold
+        );
+        _shaderProgram->setUniform(_uniformCache.fluxColorAlpha, _fluxColorAlpha);
+        _shaderProgram->setUniform(
+            _uniformCache.fluxColorAlphaIlluminance, 
+            _fluxColorAlphaIlluminance
+        );
+        _shaderProgram->setUniform(_uniformCache.earthPos, earthPos);
+        _shaderProgram->setUniform(_uniformCache.distanceThreshold, _distanceThreshold);
+        _shaderProgram->setUniform(_uniformCache.enhanceMethod, _enhancemethod);
+        _shaderProgram->setUniform(_uniformCache.flowColor, _flowColor);
+        _shaderProgram->setUniform(_uniformCache.usingParticles, _flowEnabled);
+        _shaderProgram->setUniform(_uniformCache.particleSize, _flowParticleSize);
+        _shaderProgram->setUniform(_uniformCache.particleSpacing, _flowParticleSpacing);
+        _shaderProgram->setUniform(_uniformCache.particleSpeed, _flowSpeed);
+        _shaderProgram->setUniform(
+            _uniformCache2.time,
+            global::windowDelegate->applicationTime() * -1
```

Comment:
Is this correct?

Reply:
I'm very unsure. Did not see a difference between if it was * -1 or 1 (positive), even when changing to deltaTime()

Reply:
Maybe the time uniform in the shader wasn't used?